docs/beta.md

```diff
@@ -99,7 +99,7 @@ have to run some commands manually to debug the list of beans collected, JVMs,
 etc. A typical manual call will take the following form:
 
 ```shell
-/usr/bin/java -Xmx200m -Xms50m -classpath /usr/lib/jvm/java-8-oracle/lib/tools.jar:/opt/datadog-agent6/bin/agent/dist/jmx/jmxfetch-0.17.0-jar-with-dependencies.jar org.datadog.jmxfetch.App --check <check list> --conf_directory /etc/datadog-agent/conf.d --log_level INFO --log_location /opt/datadog-agent6/bin/agent/dist/jmx/jmxfetch.log --reporter console <command>
+/usr/bin/java -Xmx200m -Xms50m -classpath /usr/lib/jvm/java-8-oracle/lib/tools.jar:/opt/datadog-agent/bin/agent/dist/jmx/jmxfetch-0.18.1-jar-with-dependencies.jar org.datadog.jmxfetch.App --check <check list> --conf_directory /opt/datadog-agent/etc/conf.d --log_level INFO --log_location /opt/datadog-agent/bin/agent/dist/jmx/jmxfetch.log --reporter console <command>
```

Comment:
can you still use `/etc/datadog-agent/conf.d` as the check conf directory? Most customers would be using these commands on Linux, makes sense to take Linux in the example.

Same thing for the change below